conbench/tests/api/test_benchmarks.py

```diff
@@ -271,12 +271,20 @@ def test_benchmark_list_filter_by_name(self, client):
 
     def test_benchmark_list_filter_by_batch_id(self, client):
         self.authenticate(client)
-        _fixtures.benchmark_result(batch_id="10")
         benchmark_result = _fixtures.benchmark_result(batch_id="20")
-        _fixtures.benchmark_result(batch_id="30")
         response = client.get("/api/benchmarks/?batch_id=20")
         self.assert_200_ok(response, [_expected_entity(benchmark_result)])
 
+    def test_benchmark_list_filter_by_multiple_batch_id(self, client):
+        self.authenticate(client)
+        benchmark_result_1 = _fixtures.benchmark_result()
+        batch_id1 = benchmark_result_1.batch_id
```

Comment:
Sorry for nit picking but the repo uses names like `base_id_1` instead of `base_id1`

Reply:
No apologies necessary, I'm grateful for this!